src/Opaleye/Internal/Sql.hs

```diff
@@ -153,10 +154,19 @@ aggregate aggrs s = SelectFrom $ newSelect { attrs = SelectAttrs
 aggrExpr :: Maybe (HPQ.AggrOp, [HPQ.OrderExpr], HPQ.AggrDistinct) -> HPQ.PrimExpr -> HPQ.PrimExpr
 aggrExpr = maybe id (\(op, ord, distinct) e -> HPQ.AggrExpr distinct op e ord)
 
-order :: [HPQ.OrderExpr] -> Select -> Select
-order oes s = SelectFrom $
-    newSelect { tables = [s]
-              , orderBy = map (SD.toSqlOrder SD.defaultSqlGenerator) oes }
+distinguishAndOrder :: Maybe (NEL.NonEmpty HPQ.PrimExpr) -> [HPQ.OrderExpr] -> Select -> Select
```

Comment:
Can you rename this `distinctOnOrderBy` for consistency?

Reply:
Absolutely.